pyproject.toml

```diff
@@ -160,8 +160,6 @@ types-setuptools = { version = "^57.4.2", optional = true }
 types-six = { version = "^1.16.2", optional = true }
 types-termcolor = { version = "^1.1.2", optional = true }
 types-psutil = { version = "^5.8.13", optional = true }
-# test framework dependencies
```

Comment:
Removing this will break the CI

Reply:
I adjusted the integration test GH action to lazily install `docker-compose` in non-default environments now. I think that should work since the CI only runs non-default environment tests under Python 3.8 where PyYAML shouldn't be broken in the first place.